test/Interop/Cxx/class/method/Inputs/methods.h

```diff
@@ -0,0 +1,30 @@
+#ifndef TEST_INTEROP_CXX_CLASS_METHOD_METHODS_H
+#define TEST_INTEROP_CXX_CLASS_METHOD_METHODS_H
+
+struct NonTrivialInWrapper {
+  int value;
+
+  ~NonTrivialInWrapper() { }
+};
+
+struct HasMethods {
+  void nonConstMethod() { }
+  void constMethod() const { }
+
+  int nonConstPassThrough(int a) { return a; }
+  int constPassThrough(int a) const { return a; }
```

Comment:
Not sure what "passthrough" refers to here.  Is it the argument or the fact that they are inline?

Reply:
Yeah, just passing through the arg.